flang/test/Driver/msvc-dependent-lib-flags.f90

```diff
@@ -0,0 +1,36 @@
+! RUN: %flang -### --target=aarch64-windows-msvc -fuse-ld= %S/Inputs/hello.f90 -v 2>&1 | FileCheck %s --check-prefixes=MSVC

```

Comment:
Could you remind the rationale behind an empty `-fuse-ld=`?

Reply:
I don't know that there is one honestly, I copied these build lines from another file! Happy to remove it

Reply:
Lets remove it then.